ringpop_stub_test.go

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
Better to name this file 'ringpop_stub.go'? The _test suffix makes it sound like it contains tests.